go/vt/vtgate/evalengine/expressions.go

```diff
@@ -67,14 +69,44 @@ type (
 		Offset    int
 		Collation collations.ID
 	}
+	Tuple []Expr
 )
 
+// Evaluate implements the Expr interface
+func (t Tuple) Evaluate(env ExpressionEnv) (EvalResult, error) {
+	var res EvalResult
+	res.typ = querypb.Type_TUPLE
+	for _, expr := range t {
+		evalRes, err := expr.Evaluate(env)
+		if err != nil {
+			return EvalResult{}, err
+		}
+		res.tupleResults = append(res.tupleResults, evalRes)
+	}
+	return res, nil
+}
+
+// Type implements the Expr interface
+func (t Tuple) Type(env ExpressionEnv) (querypb.Type, error) {
+	return querypb.Type_TUPLE, nil
+}
+
+// String implements the Expr interface
+func (t Tuple) String() string {
+	var stringSlice []string
+	for _, expr := range t {
+		stringSlice = append(stringSlice, expr.String())
+	}
+	return "(" + strings.Join(stringSlice, ",") + ")"
+}
```

Comment:
small nit, but can we also move this down in the file, where the other implementations are

Reply:
Sure